packages/filecoin-wallet-provider/src/wallet-sub-provider.ts

```diff
@@ -0,0 +1,7 @@
+import { LotusMessage, SignedLotusMessage } from '@glif/filecoin-message'
+
+export interface WalletSubProvider {
+  newAccount?(): Promise<string>
```

Comment:
Can take this out, but I'm happy to be convinced otherwise

Reply:
Better be present. I feel, like it is possible at some point in future to have a provider that manages HD Keys.